codebase/superdarn/src.lib/tk/elevation.1.0/src/elev_goose.c

```diff
@@ -20,17 +20,18 @@ You should have received a copy of the GNU General Public License
 along with this program. If not, see <https://www.gnu.org/licenses/>.
 
 Modifications:
+  E.G.Thomas 2021-08: added support for bmoff parameter
 */ 
 
 
 
 #include <math.h>
 #include <stdio.h>
 
-#include "fitblk.h"
 #include "rmath.h"
+#include "fitblk.h"
 
-double elev_goose(struct FitPrm *prm, double phi0) {
+double elev_goose(struct FitPrm * prm, double phi0) {
```

Comment:
```suggestion
double elev_goose(struct FitPrm *prm, double phi0) {
```

This seems like an unnecessary stylistic change from the rest of the RST.